docs/database_schema_diagrams/source/entity_network_details.json

```diff
@@ -1,6 +1,6 @@
 {
-  "tooltip": "Tables specifically related to core entities",
-  "_comment": "It shows tables unique to some entities, that is, beyond aliases, collections, edits, relationships, subscriptions, ratings, tags.",
+  "tooltip": "Tables specifically related to some primary entities",
```

Comment:
"some primary"? Should it be "all primary" and we can add any missing ones, or?

Reply:
Correct, I dropped “some” and tried to make it more clear with https://github.com/metabrainz/musicbrainz-server/pull/2987/commits/d34e4232a251e0652995c1ce7138e48227438a74#diff-823a23760774550d2454cd9e3f8a20311f20a85cf6672dcdc93cdbc00fc03ae7